apps/frontend/src/app/App.tsx

```diff
@@ -70,8 +70,10 @@ function ScrollTop({ children }: { children: React.ReactElement }) {
 
 function App() {
   const dbIndex = parseInt(localStorage.getItem('dbIndex') || '1')
-  const [databases, setDatabases] = useState(() =>
-    ([1, 2, 3, 4] as const).map((index) => {
+  const [databases, setDatabases] = useState(() => {
+    localStorage.removeItem('GONewTabDetection')
```

Comment:
semi-nit: this should be a const stored somewhere

Reply:
i figured adding a new file + 2 imports for a const might be overkill

Reply:
All our db keys should have consts. We can leave it for another day then